### PR TITLE
(MAINT) Update Changelog to KAC Format

### PR DESCRIPTION
**Pull request by RandomNoun7**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
 
-## 2017-12-08 - Supported Release 2.1.3
+- Metadata for supporting Windows Server 2016 ([MODULES-4271](https://tickets.puppetlabs.com/browse/MODULES-4271))
 
-### Summary
+### Fixed
 
-Small release with bug fixes.
+- Upgrade message is improved to make .NET Framework requirements clearer when running PowerShell 2.0 ([MODULES-7011](https://tickets.puppetlabs.com/browse/MODULES-7011))
+- Fix timeout handling when the user specified a timeout parameter value of `0` ([MODULES-7018](https://tickets.puppetlabs.com/browse/MODULES-7018))
```
Comment (michaeltlombardi) [inline CHANGELOG.md]: I'd clarify here that is assigns a sane default of 300 seconds to be clear about the change for a user.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,48 +1,49 @@
...
 
-### Bug Fixes
+## [2.1.4] - 2017-03-29
 
-- Fix timeouts and zombie process creation ([MODULES-4748](https://tickets.puppetlabs.com/browse/MODULES-4748))
-- Update PowerShell executable name for experimental cross-platform / PowerShell 6 support ([MODULES-6081](https://tickets.puppetlabs.com/browse/MODULES-6081))
+### Fixed
 
-## 2017-07-27 - Supported Release 2.1.2
+- Unabled to start pipes server in PowerShell process on some Windows 2008r2 images ([MODULES-6927](https://tickets.puppetlabs.com/browse/MODULES-6927))
```
Comment (michaeltlombardi) [inline CHANGELOG.md]: `Unabled to`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,48 +1,49 @@
...
 
-### Summary
+## [2.1.3] - 2017-12-08
 
-Small release with bugs fixes.
+### Fixed
 
-### Bug Fixes
+- Fix timeouts and zombie process creation ([MODULES-4748](https://tickets.puppetlabs.com/browse/MODULES-4748))
+- Update PowerShell executable name for experimental cross-platform / PowerShell 6 support ([MODULES-6081](https://tickets.puppetlabs.com/browse/MODULES-6081))
```
Comment (michaeltlombardi) [inline CHANGELOG.md]: The way this is written sounds more like a feature than a bug - change 'update' to 'corrected the' - this should make it clearer that we're now using the proper exe name.

**Comment by michaeltlombardi** [comment]
I'm	not	particularly concerned with rewriting stuff from many moons ago, but I think we may want to improve the wording for a few recent entries. Otherwise, 👍

**Comment by RandomNoun7** [comment]
@michaeltlombardi Edits in place.

**Pull request merged**